scripts/jenkins/ardana/ansible/roles/ardana_update/tasks/main.yml

```diff
@@ -15,6 +15,18 @@
 #
 ---
 
+- name: Get list of non-deployer nodes
+  shell: |
+    deployer=`grep -A 1 ARD-SVC--first-member hosts/verb_hosts | tail -1`
```

Comment:
You should be able to use the ansible command's --list-hosts mechanism to get this information, and the list of resource nodes, and in fact you should be able to get the list of resource nodes with a single ansible command using ansible host specification syntax, e.g.

    ansible 'resources:&ARD-SVC--first-member' --list-hosts